app/full_text_search.py

```diff
@@ -106,16 +109,78 @@ def search(repo, query_txt, max_results):
             results.append(result)
     return results
 
-def create_document(**kwargs):
+
+def create_jp_name_document(**kwargs):
+    """
+    Creates document(romanized jp name data) for full text search.
+    It should be called in create_document method.
+    """
+    fields = []
+    romanized_names_list = []
+    for field in kwargs:
+        romanized_japanese_name =\
+                script_variant.romanize_japanese_name_by_name_dict(kwargs[field])
+        romanized_names_list.append(romanized_japanese_name)
+        fields.append(
+            appengine_search.TextField(name=field+'_romanized_by_jp_name_dict',
+                                       value=romanized_japanese_name)
+        )
+    romanized_jp_names =\
+            ':'.join([name for name in romanized_names_list if name])
+    fields.append(appengine_search.TextField(name='romanized_jp_names',
+                                             value=romanized_jp_names))
+    return fields
+
+
+def create_jp_location_document(**kwargs):
+    """
+    Creates document(romanized jp location data) for full text search.
+    It should be called in create_document method.
+    """
+    fields = []
+    for field in kwargs:
+        romanized_japanese_location =\
```

Comment:
Let's use () instead of \

romanized_japanese_location = (
    script_variant.romanize_japanese_location(kwargs[field]))